src/Provider/Provider.Web/ViewModels/VacancyPreview/VacancyPreviewViewModel.cs

```diff
@@ -466,5 +470,17 @@ public enum VacancyTaskListSectionState
         InProgress,
         Completed
     }
+
+    public class MustBeTrueAttribute : ValidationAttribute
```

Comment:
Same as above, can be removed but if we were using then should be in a shared place and not nested

Reply:
Removed in latest.